torch/_dynamo/variables/builtin.py

```diff
@@ -831,6 +856,22 @@ def constant_fold_handler(tx: "InstructionTranslator", args, kwargs):
 
             handlers.append(constant_fold_handler)
 
+        elif obj.is_unfoldable_op():
+            builder = SourcelessBuilder.create
+            def unfoldable_handler(tx: "InstructionTranslator", args, kwargs):
+                try:
+                    res = fn(
+                        *[x.as_python_constant() for x in args],
+                        **{
+                            k: v.as_python_constant() for k, v in kwargs.items()
+                        },
+                    )
+                except Exception as exc:
```

Comment:
Should this really be a generic exception? Not even, what exceptions are you expecting and can we make it more narrow?

Reply:
Some type of exceptions can come user's side, I can think of 2 right now: incorrect type and/or number of arguments i.e. `operator.delitem(2, 3)` and indexing errrors `operator.delitem([1,2,3], 12)`. I though other sorts might come from user side, also `operator.call` might generate some types. What do you think? I can investigate better what sort of exceptions come from these 3 ops and narrow down surely.